src/Graphics/OpenGLContext/mupen64plus/mupen64plus_DisplayWindow.cpp

```diff
@@ -59,7 +59,9 @@ void DisplayWindowMupen64plus::_setAttributes()
 	CoreVideo_GL_SetAttribute(M64P_GL_CONTEXT_MAJOR_VERSION, 3);
 	CoreVideo_GL_SetAttribute(M64P_GL_CONTEXT_MINOR_VERSION, 2);
 #else
-	// Do nothing
+	CoreVideo_GL_SetAttribute(M64P_GL_CONTEXT_PROFILE_MASK, M64P_GL_CONTEXT_PROFILE_CORE);
+	CoreVideo_GL_SetAttribute(M64P_GL_CONTEXT_MAJOR_VERSION, 3);
+	CoreVideo_GL_SetAttribute(M64P_GL_CONTEXT_MINOR_VERSION, 3);
```

Comment:
These changes should work for OS_MAC_OS_X, so please just remove OS_MAC_OS_X section above.